lib/router/mbaas.js

```diff
@@ -6,10 +6,10 @@ var userAuth = require('./mbaas-auth');
 var sessionMiddleware = require('./mbaas-session-middleware');
 var _ = require('lodash');
 
-function initRouter(mediator, authResponseExclusionList) {
+function initRouter(mediator, authResponseExclusionList, expressSessionMiddleware) {
   var router = express.Router();
 
-  router.all('/auth', function(req, res) {
+  router.all('/auth', function(req, res, next) {
```

Comment:
Don't think `next` is needed here since `expressSessionMiddleware` is being called as a standalone function. This triggers a lint error.

We should either remove `next` or use `expressSessionMiddleware` as the last handler in the chain.
There doesn't seem to be any need for it to be run in the current order so `router.all('/auth', expressSessionMiddleware, function(req, res) {...}` could do the trick.